File: compiler/examples/native/ios/UserActions.swift
/*
 * Copyright (c) Dotcorr Studio. and affiliates.
 *
 * Licensed under the PolyForm Noncommercial License 1.0.0.
 * Commercial use requires a license from DotCorr.
 */
import Foundation
enum UserActions {
    static func a_change(_ model: AppModel) {
        model.s_message = "Updated by Swift on " + ProcessInfo.processInfo.operatingSystemVersionString
    }
}

File: compiler/examples/native/android/UserActions.java
/*
 * Copyright (c) Dotcorr Studio. and affiliates.
 *
 * Licensed under the PolyForm Noncommercial License 1.0.0.
 * Commercial use requires a license from DotCorr.
 */
package com.dotcorr.escape;
public final class UserActions {
    public static void a_change(AppModel model) {
        model.s_message = "Updated by Java on Android " + android.os.Build.VERSION.RELEASE;
    }
}
